canonical-web-and-design/ubuntu.com
# Update code-snippets to vanilla-3.0 patterns

## Done

- This update `p-code-copyable` to be a regular `p-code-snippet` as we don't use copyable code anymore.
- It also removes a JS file called scratch.js that was responsible for copying the code

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Go to each of the pages effected and check the code snippets look good.

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4860

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)
